4:3 aspect ratio tidy-up

CrashCringle12 opened the pull request:
4:3 was a bit cluttered together, both in Regular and Marathon mode. This commit aims to tidy it up a bit.

### Before:
![4-3Before](https://user-images.githubusercontent.com/30600688/209750370-5d392488-13fc-4642-be07-e1ad75356d17.png)
![image](https://user-images.githubusercontent.com/30600688/209750629-b5764ea2-6b7e-4f08-a278-9ec243b875c5.png)

<br><br>

### After:
![4-3After](https://user-images.githubusercontent.com/30600688/209750392-e3939c29-8830-4634-b01a-00caa9e763db.png)
![image](https://user-images.githubusercontent.com/30600688/209750685-968f3f4d-2813-4c85-9a89-02eade0dbd08.png)

CrashCringle12 committed:
Tidied up Music select screens for 4:3

(cherry picked from commit 305fd7c3818661285b6926f0666f8f9e1177ab4e
RE: https://github.com/CrashCringle12/Simply-Love-SM5/issues/14)

teejusb commented:
Can you highlight the main differences? All I can tell is that you added some spacing on the left edge?

CrashCringle12 commented:
Sure (the changes were made specifically for 4:3 support and don't modify widescreen aspect ratios)

- Resized Banner, DensityGraph, and StepArtist displays (Made them smaller so they could better fit)
- Moved each of the above further to the right 
- Reduced the size of the text in PaneDisplay and repositioned each of the columns (Were previously close to the edge and size was large making it cluttered on the display)
- Resized coursemode display and realigned.
- Resized groovestats loading icon and moved to a visible location (In between banner and musicwheel)

teejusb commented:
I'll test this out on my dedi to see what it plays like and report back.

quietly-turning commented:
> Reduced the size of the text in PaneDisplay

This part can be tricky.

The `local text_zoom = WideScale(0.8, 0.9)` that I used in PaneDisplay.lua was based on my decision to support stock monitors from traditional DDR cabs, which (if I understand correctly) run at 640x240 @ 15khz refresh, interlaced to give the appearance of what we'd think of as 640x480.

It's *tricky* because I don't think LCD screens can properly simulate what that really looks like, which made first-hand testing impossible.  By having back-and-forth discussions with community members who had that hardware, I arrived at the fuzzy understanding that SL's Miso glyphs couldn't be zoomed any smaller than `0.8` without becoming introducing legibility issues.

For example, the list of player modifiers and stepartist text on ScreenEvaluation are both hardcoded to `0.7` zoom, and both are difficult to read in this photo:
![6a49a12b-038a-4762-9d64-9e9878966862](https://user-images.githubusercontent.com/1253483/218326506-00ac25a0-c6a8-4b4a-92d9-2b915ade358d.jpg)

With regard to stock DDR cab monitors, I treated each SL screen as a conscious design decision and asked, "how important is it that this text be legible?"  If it was important be legible, I didn't go under `0.8` zoom, based on the Miso glyphs in SL at that time.

I understand the glyphs themselves have changed since then, and that the goals/targeted hardware to support may also have changed.  But, this was my reasoning for font zooms back then. :)

CrashCringle12 commented:
maybe will revisit this in the future but not now
